.eslintrc.json

```diff
@@ -10,10 +10,7 @@
         }
     },
     "parser": "babel-eslint",
-    "plugins": [
-        "react",
-        "mocha"
-    ],
+    "plugins": ["react", "mocha"],
```

Comment:
why the changes on this file? do we really need them?

Reply:
Looks like my editor formatted it. I'll revert